__tests__/app.test.js

```diff
@@ -105,4 +104,54 @@ describe('GET /api/articles', () => {
         })
     });
 
-});
\ No newline at end of file
+});
+
+describe('GET /api/articles/:article_id/comments', () => {
+    test('200: return array of comments for the valid article_id', () => {
+        return request(app)
+        .get('/api/articles/3/comments')
+        .expect(200)
+        .then(({ body }) => {
+            const {comments} = body
+            expect(comments).toBeSortedBy("created_at", {
+                descending: true})
+            comments.forEach((comment) => {
```

Comment:
You should assert the length of the comments, look at the test data, and see how many there should be to ensure that your forEach and sorted are actually checking a populated array